flink-python/src/main/java/org/apache/flink/client/python/PythonDriver.java

```diff
@@ -59,6 +61,8 @@ public static void main(String[] args) {
 			System.exit(1);
 		}
 
+		Configuration config = ExecutionEnvironment.getExecutionEnvironment().getConfiguration();
```

Comment:
Does this work for streaming job?

Reply:
Yes, the configurations of streaming and batch environments are always set at the same time.